### ISSUE TITLE
Swedish language short-code inconsistency

### ISSUE THREAD

**Issue by sakjur** [googlei18n/libphonenumber#1240]
In `resources/geocoding` the directory `se` (Sweden's CC) contains localization for area names in Swedish for Finland whereas the directory `sv` (two-letter code for the language Swedish) contains information for area names in Swedish for Sweden.

**Steps to reproduce**

Test environment: David Drysdale's Python 3 port [python-phonenumbers](https://github.com/daviddrysdale/python-phonenumbers) (However, the error was detected by looking manually through the resources-files and ought to be reproducible on every platform)

Numbers used in tests: SE +46313650000 and FI +358947001

Use the geocoder to get the description for number for the Swedish and Finnish number.

```
import phonenumbers
from phonenumbers import geocoder

locales = ['se', 'sv', 'en']
numbers = ['+46313650000', '+358947001']

for number in numbers:
    parsed_number = phonenumbers.parse(number, None)
    for locale in locales:
        description = geocoder.description_for_number(parsed_number, locale)
        print("%s %s %s" % (locale, number, description))

```

**Result**

```
se +46313650000 Gothenburg
sv +46313650000 Göteborg
en +46313650000 Gothenburg
se +358947001 Helsingfors
sv +358947001 Helsinki
en +358947001 Helsinki
```

Expected (without adding Northern Sami):

```
se +46313650000 Gothenburg
sv +46313650000 Göteborg
en +46313650000 Gothenburg
se +358947001 Helsinki
sv +358947001 Helsingfors
en +358947001 Helsinki
```

or (with support for Northern Sami)

```
se +46313650000 Göteborg
sv +46313650000 Göteborg
en +46313650000 Gothenburg
se +358947001 Helsset
sv +358947001 Helsingfors
en +358947001 Helsinki
```

or (without support for Northern Sami, aliasing `se` to `sv` for compatability and convenience)

```
se +46313650000 Göteborg
sv +46313650000 Göteborg
en +46313650000 Gothenburg
se +358947001 Helsingfors
sv +358947001 Helsingfors
en +358947001 Helsinki
```

**Comment by padmaksha** [comment]
@sakjur 

Thanks for taking the time to improve libphonenumber!
This looks like a bug. We'll fix it in an upcoming release. 
We will move the Finish data to sv directory too.. 

Thanks

**Comment by lararennie** [comment]
Fixed in v7.7.3
